src/github/pull-request.ts

```diff
@@ -111,3 +95,29 @@ export default async (
 		jiraResponse?.status
 	);
 };
+
+const updateGithubIssues = async (github: GitHubClient | GitHubAPI, context: CustomContext, util, pullRequest) => {
+	const linkifiedBody = await util.unfurl(pullRequest.body);
+	if (!linkifiedBody) {
+		return;
+	}
+
+	context.log("Updating pull request");
+	const editedPullRequest = context.issue({
+		body: linkifiedBody,
+		id: pullRequest.id
+	});
+
+	const { body, id, owner, repo, number } = editedPullRequest;
```

Comment:
yeah, you're already creating the object here really.  Cut out the middle man ;)